HCore-Scheduling/HCore-Scheduling.csproj

```diff
@@ -7,9 +7,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Quartz" Version="3.4.0" />
-    <PackageReference Include="Quartz.Serialization.Json" Version="3.4.0" />
-	<PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="17.2.32">
+    <PackageReference Include="Quartz" Version="3.8.0" />
```

Comment:
Tested?

Reply:
Yes,
When PortalsAPI-B bootstraps it uses SchedulingProviderImpl, which calls                 `scheduler.ScheduleJob(jobInstance, jobTrigger).Wait();`